src/common/hooks/usePaidAccount.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react'
+
+import { ManualAccountType } from '../enums'
+
+export const usePaidAccount = () => {
+  const [isPaid, setIsPaid] = useState(false)
+  const [deleteCookie, setDeleteCookie] = useState(false)
+
+  useEffect(() => {
+    if (deleteCookie) {
+      document.cookie = `${ManualAccountType.paymentCookies}; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;`
+    }
+    const cookie = document.cookie.split('; ')
```

Comment:
Множественное число лучше будет использовать
```suggestion
    const cookies = document.cookie.split('; ')
```